doc/source/whatsnew/v0.15.0.rst

```diff
@@ -792,13 +791,13 @@ Other notable API changes:
 
   This is now the correct behavior
 
-  .. ipython:: python
+  ..  ipython:: python
```

Comment:
revert the addition of the space here (sphinx is very sensitive to this kind of things ..)